setup.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+
+from setuptools import setup
+
```

Comment:
Should add https://github.com/jmathai/elodie/pull/345/files#diff-2eeaed663bd0d25b7e608891384b7298R11-R12 to the setup, unless pbr already does that?